src/main/java/Deadline.java

```diff
@@ -0,0 +1,63 @@
+public class Deadline extends TaskList{
+    private static final String TASKTITLE ="D";
```

Comment:
Inconsistent format for naming constants, some have _ in between words but not this time.

Reply:
Noted!!!